Logos/varmour-logo.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Logo is not sized properly, viewbox is much larger than content so logo appears tiny.

Reply:
Will be updated later